index.html

```diff
@@ -189,111 +223,118 @@ <h4 class="colorh4">CANOPY</h4>
-                </li>
-                <li>
-                    <a title="The link will take you to my twitter profile" href="https://github.com/MPDADDY">
-                        <i class="fa fa-twitter social"></i>
-                    </a>
-                </li>
-                <li>
-                    <a title="The link will take you to my instagram profile"
-                        href="https://www.instagram.com/mpdaddympd/">
-                        <i class="fa fa-instagram social"></i>
-                    </a>
-                </li>
-                <li>
-                    <a title="The link will take you to my linkin profile"
-                        href="https://www.linkedin.com/in/anold-mwawasi">
-                        <i class="fa fa-linkedin social"></i>
-                    </a>
-                </li>
-            </ul>
-            <div class="getresume">
-                <a title="get my resume" href="#">Get my resume</a>
-            </div>
-        </article>
-        <article id="what_i_do" class="padding">
+        <div class="aboutmyself ">
+            <article class="padding">
+                <h1>About Myself</h1>
+                <p>I'm a software developer! I can help you build a product , feature or website Look through some of my
+                    work
+                    and experience! If you like what you see and have a project you need coded, don't hesitate to
+                    contact
+                    me.
+                </p>
+                <h4>LET'S CONNECT</h4>
+                <ul>
+                    <li>
+                        <a title="The link will take you to my facebook profile"
+                            href="https://www.facebook.com/Mpdaddy/">
+                            <i class="fa fa-facebook social"></i>
+                        </a>
+                    </li>
+                    <li>
+                        <a title="The link will take you to my twitter profile" href="https://github.com/MPDADDY">
+                            <i class="fa fa-twitter social"></i>
+                        </a>
+                    </li>
+                    <li>
+                        <a title="The link will take you to my instagram profile"
+                            href="https://www.instagram.com/mpdaddympd/">
+                            <i class="fa fa-instagram social"></i>
+                        </a>
+                    </li>
+                    <li>
+                        <a title="The link will take you to my linkin profile"
+                            href="https://www.linkedin.com/in/anold-mwawasi">
+                            <i class="fa fa-linkedin social"></i>
+                        </a>
+                    </li>
+                </ul>
+                <div class="getresume">
+                    <a title="get my resume" href="#">Get my resume</a>
+                </div>
+            </article>
+            <article id="what_i_do" class="padding">
```

Comment:
- [ ] kindly consider giving this section the same margin and padding ,width and fort the `get my resume` button give it the same color as the one in Figma to implement exactly the one in the design consider checking on the below screenshots to understand what you have to update clearer
- your design 👇
<img width="770" alt="Screen Shot 2023-05-12 at 15 31 51" src="https://github.com/MPDADDY/portfolio/assets/60146030/d4417ef1-c487-4a31-8bfd-4aa060c48910">

- Figma design 👇 
<img width="808" alt="Screen Shot 2023-05-12 at 15 32 59" src="https://github.com/MPDADDY/portfolio/assets/60146030/5aacbe5b-a1fa-4871-8ed6-bfb5c4212aba">